src/utils/data/uid.ts

```diff
@@ -7,23 +7,23 @@ type UIDNames =
     | 'texture'
     | 'textureSource'
     | 'textureResource'
-    | 'batcher'
-    | 'graphicsContext'
-    | 'graphicsView'
-    | 'graphicsPath'
-    | 'fillGradient'
-    | 'fillPattern'
-    | 'meshView'
-    | 'renderable'
-    | 'buffer'
-    | 'bufferResource'
+    | 'batcher' //
+    | 'graphicsContext' //
+    | 'graphicsView' //
```

Comment:
What is this for?

Reply:
any // we can remove - but did not want to muddy the PR!